json/src/scheme/params.rs

```diff
@@ -1,80 +0,0 @@
-// Copyright 2018-2020 Kodebox, Inc.
```

Comment:
I thought about whether remove this or not.

We may need this code to serialize Consensus?  params.

We don't need it now. This file is in scheme folder, We may rewrite similar code easily.
So I removed this.

Reply:
ok.